src/DataWrangling/restoring.jl

```diff
@@ -79,21 +78,21 @@ function set!(fts::ECCOFieldTimeSeries)
 end
 
 """
-    ECCOFieldTimeSeries(metadata::ECCOMetadata [, arch_or_grid=CPU() ];
+    FieldTimeSeries(metadata::Metadata [, arch_or_grid=CPU() ];
                         time_indices_in_memory = 2,
                         time_indexing = Cyclical(),
                         inpainting = nothing,
                         cache_inpainted_data = true)
 
-Create a field time series object for ECCO data.
+Create a field time series object for obs/reanalysis data.
```

Comment:
don't wanna mention "obs/reanalysis", right?
perhaps something along the lines of:

```suggestion
Create a FieldTimeSeries from a dataset that corresponds to `metadata`..
```